### ISSUE TITLE
Build failing on FreeBSD 11-CURRENT

### ISSUE THREAD

**Issue by rbgarga** [esnet/iperf#413]
I was trying to build iperf on FreeBSD 11-CURRENT and got an error, complaining about TAILQ_END missing:
```
cc -DHAVE_CONFIG_H -I.    -g -O2 -pipe  -fstack-protector -fno-strict-aliasing -Wall -MT iperf3-main.o -MD -MP -MF .deps/iperf3-main.Tpo -c -o iperf3-main.o `test -f 'main.c' || echo './'`main.c
mv -f .deps/iperf3-main.Tpo .deps/iperf3-main.Po
/bin/sh ../libtool  --tag=CC    --mode=link cc -g -O2 -pipe  -fstack-protector -fno-strict-aliasing -Wall  -g -fstack-protector -o iperf3 iperf3-main.o libiperf.la 
libtool: link: cc -g -O2 -pipe -fstack-protector -fno-strict-aliasing -Wall -g -fstack-protector -o .libs/iperf3 iperf3-main.o  ./.libs/libiperf.so -Wl,-rpath -Wl,/usr/local/lib
./.libs/libiperf.so: undefined reference to `TAILQ_END'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
*** Error code 1
```
After investigate I saw iperf ships its own queue.h, but, for some reason, it's including sys/queue.h from /usr/include and ignoring bundled queue.h since it uses the same header guard name _SYS_QUEUE_H.

FreeBSD 11 sys/queue.h doesn't have TAILQ_END defined, then it causes linker error.

To fix it on pfSense, what I did was replace single place using TAILQ_END by NULL, as you can see at:

https://github.com/pfsense/FreeBSD-ports/commit/a3aea4106d1e87862214ba77e36e6f2a33f732e9

**Comment by bmah888** [comment]
Hrm.  I've been seeing this build error from the FreeBSD Ports build cluster, but didn't get a chance to investigate it yet.  Your fix looks pretty good to me, and it'd probably apply well to the various platforms supported by iperf3.

Separately from that, I'm actually not sure why iperf3 includes its own copy of queue.h, and why it apparently gets used on some platforms and not others.

Thanks for the pointer to the patch...I'll commit it after a bit of cross-platform testing.

**Comment by bmah888** [comment]
Merged to 3.1-STABLE branch.
